apps/dashboard/src/components/cursor-pagination.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@/components/primitives/button';
+import { ChevronLeftIcon, ChevronRightIcon } from '@radix-ui/react-icons';
+
+interface CursorPaginationProps {
+  hasNext: boolean;
+  hasPrevious: boolean;
+  onNext: () => void;
+  onPrevious: () => void;
+  onFirst: () => void;
+}
+
+export function CursorPagination({ hasNext, hasPrevious, onNext, onPrevious, onFirst }: CursorPaginationProps) {
+  return (
+    <div className="bottom-0 mt-auto border-t border-t-neutral-200 bg-white py-3">
+      <div className="flex items-center justify-center px-6">
+        <div className="border-input inline-flex items-center rounded-lg border bg-transparent">
```

Comment:
Something is off here. We have a flex that hosts an inline-flex. I believe we can use only one div instead of two, one wrapping the other.